src/EventApi.php

```diff
@@ -243,6 +243,17 @@ public function persist(CalendArtAbstractEvent $event)
             $method = 'post';
         }
 
+        // manage the options now
+        // should we send notifications to the attendees ?
+        if (isset($options['sendNotifications'])) {
```

Comment:
I test the value of the option here https://github.com/Calendart/GoogleAdapter/pull/13/files/29f9eb4b0d170924d71304671bd082b877bb332e#diff-1369ed56bd7644f751c9b9838ed842a9R252